docker-compose.yml

```diff
@@ -4,90 +4,83 @@ x-labels: &k8s-labels
   kompose.image-pull-secret: pullsecret
 
 services:
-  webcsd:
+  csd-platform:
+    environment:
+      - PublicUri=${PUBLIC_URI}
+    image: ccdcrepository.azurecr.io/onsite/ccdc-web-platform:4.0.0
+    restart: unless-stopped
+    ports:
+      - 443:8443
+    labels:
+      <<: *k8s-labels
+      kompose.service.type: LoadBalancer
+      kompose.service.expose: "true"
+    depends_on:
+      - ccdc-identity
+      - webcsd
+      - csd-python-gateway
+
+  csd-python-gateway:
+    environment:
+      - PlatformSecurity__Issuer=${PUBLIC_URI}
+      - CCDC_LICENSING_CONFIGURATION=${CCDC_LICENSING_CONFIGURATION}
+    image: ccdcrepository.azurecr.io/onsite/ccdc-python-gateway:4.0.0
+    restart: unless-stopped
+
+  ccdc-identity:
+    image: ccdcrepository.azurecr.io/onsite/ccdc-identitygateway:4.0.0
     environment:
-      - CsdRepository__ServiceLocation=webcsdbackend
-      - CspRepository__ServiceLocation=webcsd-theory
+      - "ConnectionStrings__DefaultConnection=${CSD_DB_CONNECTIONSTRING};SearchPath=identity;"
```

Comment:
Is this right? identity will be in the same database as CSD Data?

Reply:
Nope have updated :)